src/sections/footer.html

```diff
@@ -1,4 +1,30 @@
-<div>
-    <h1>Footer</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<!-- Parent -->
+<div class="bg-webGrey px-5 lg:px-9 pt-20 pb-9">
+  <!-- Image   -->
+  <div class="flex justify-center w-full">
+    <img src="/images/red-shape.svg" alt="" class="md:w-[50%] w-[100%]" />
+  </div>
+  <div
+    class="flex md:flex-row flex-col lg:gap-0 gap-6 justify-between md:flex-wrap lg:flex-nowrap pt-20 lg:pt-36 font-medium text-base md:text-lg"
+  >
+    <div class="grow">
+      <p>Agentur Baumeister</p>
+      <p>Böckhstraße 13</p>
+      <p>10967 Berlin</p>
```

Comment:
For small text, it would be better to use a `<span>` instead of a `<p>` because a paragraph adds unnecessary block-level formatting and spac
![Screenshot Capture - 2024-05-30 - 19-23-10](https://github.com/Kamikazi-Deborah/group-one-web-project/assets/101483232/986374e5-2186-4e25-8ec6-6cca470070f7)
ing, whereas a `<span>` allows for inline text without extra spacing.

![Screenshot Capture - 2024-05-30 - 19-23-41](https://github.com/Kamikazi-Deborah/group-one-web-project/assets/101483232/264f3c81-c30a-4fb9-9a6f-0e5530c5933a)

Check well the fontsize and alignment here.